src/constants.ts

```diff
@@ -0,0 +1,23 @@
+export const LOCAL_API_ENDPOINT = 'http://localhost:8000/'
```

Comment:
Should this also have no trailing slash?

Reply:
it had a trailing slash earlier so I kept it, but should be fine either way probably. will test rq locally